api/v1beta1/common_types.go

```diff
@@ -64,3 +73,54 @@ type PasswordSelector struct {
 	// Service - Selector to get the watcher service user password from the Secret
 	Service string `json:"service"`
 }
+
+// WatcherContainerImages
+type WatcherContainerImages struct {
```

Comment:
I have the doubt if we should generalize this more as it may be good to use it for other common parameters for all the 2nd level CRs but not in the initial one. It could be something like WatcherSubCrsCommon ?

I'm not sure if having a reusable struct for only one parameter makes sense.

Reply:
We will see it in future if we can add more params there.